test/FluentMigrator.Tests/Integration/Migrations/Postgres/IndexMigration.cs

```diff
@@ -0,0 +1,47 @@
+#region License
+// Copyright (c) 2020, FluentMigrator Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+using FluentMigrator.Postgres;
+
+namespace FluentMigrator.Tests.Integration.Migrations.Postgres
+{
+    [Migration(20201127080600)]
+    public class IndexMigration : Migration
```

Comment:
How can I run/test this migration?

Reply:
You can activate the integration tests for Postgres by configuring a connection string in the `appsettings.json` for the test project and enabling it.